versioned_docs/version-4.0.0/tributech_node/Webhook_integration.md

```diff
@@ -0,0 +1,176 @@
+
+### Edit
+If we want to update the settings for a Webhook subscription we can do that via the three dots in the actions column. 
+
+![Tributech Node - Webhooks Overview Edit](./img/Webhook_Overview_Edit.png) 
+
+This way the events, webhook url or secret can be adjusted for the current Subscription.
+
+![Tributech Node - Webhooks Edit](./img/Webhook_Edit.png)
+
+### Deactivate
+We can disable the previously created Webhook by clicking the three dots in the action colum and choosing `Toggle Active` button to pause the notifications.
+
+![Tributech Node - Webhooks Deactivate](./img/Webhook_Overview_ActivateToggle.png) 
+
+### Deletion
+If we want to delete a Webhook subscription we can do that via the three dots in the actions column. 
+
+![Tributech Node - Webhooks Overview Edit](./img/Webhook_Overview_Delete.png) 
+
+## Webhook Events
+
+### Headers
+Webhooks itself are HTTP objects that contain in addition to default HTTP information Tributech specific attributes which we will discuss in the following section. The following Headers are present in the Webhook request: 
+
+| Attribute | Description | Sample Value |
+|--|--|--|
+| x-tributech-correlationid |  A trace id to track the data through our node | 00000000-0000-0000-0000-000000000000 |
+| x-tributech-eventid |  unique identifier through the node (guid) to identify the event |  49acaa7b-fa72-4863-ab4b-7933fedeb59a |
+| x-tributech-event | event name to identify the type of event | ValueReceivedEvent |
+| x-tributech-signaturetimestamp | creation timestamp utc of the signature | 06/16/2023 05:08:46 +00:00 |
+| x-tributech-signature | signature of the webhook event |  sha256=0316443119ED851EE18AA6176E8281E6A28<br />D02A52EAF5DC85D45A6149173F412 |
+| x-tributech-timestamp | event timestamp utc |  06/16/2023 05:08:43 +00:00 |
+
+### Event Types {#event-types}
+When creating a Webhook subscription we can choose different type of events. What Event each Type represents can be taken from the Tooltip.
+Its important to note that each of the Event Types defines in the background a property `EventQoS` which will be transmitted with each
+Payload indicating the frequency of the events. This property is used by the redelivery mechanism to know how often a event should be retried to resend
+in case of an error. This value is predefined and cannot change by the user. The Webhook Events are divided into two categories : 
+1. Event QOS = 1 , high frequency events (like data received)
+2. Event QOS = 2, standard frequency events 
+
+### Error Handling
+In the case of an error during Webhook Event delivery we implemented an back-off redelivery mechanism. 
+Commonly errors occour in the following situations:
+
+- Network is not available
+- Endpoint described within the subscription is not reachable
+- Endpoint returns something else then a HTTP 200 (OK) 
+
+If an error occours we execute retries based on the formulary  `(Attempt Count + 0,7) ^ 4 + Minimum Retry Interval`.
```

Comment:
There is an exception if the event is declared as high frequency event, then the error handling is not used and the error event will be discarded